### PR TITLE
Proofreading of the webflux chapter in the reference guide

### PR DESCRIPTION
**Pull request by Buzzardo**
I proofread the WebFlux chapter.

### CODE REVIEW

File: src/docs/asciidoc/web.adoc [hunk 1]
```diff
@@ -1,25 +1,30 @@
 [[spring-web]]
-= Web
+= The Web
```
Comment (sdeleuze) [inline src/docs/asciidoc/web.adoc]: Documentation has been splitted between Web and Reactive Web stack, so not sure about "The web" which was previous unique title for both stack.

File: src/docs/asciidoc/web.adoc [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
 :doc-root: https://docs.spring.io
 :api-spring-framework: {doc-root}/spring-framework/docs/{spring-version}/javadoc-api/org/springframework
 :toc: left
 :toclevels: 2
 
-This part of the documentation covers support for web applications designed to run on a
-traditional Servlet stack (Servlet API + Servlet container).
```
Comment (sdeleuze) [inline src/docs/asciidoc/web.adoc]: As stated previously, this document now concerns only the servlet-based stack, so this statement is on purpose.

File: src/docs/asciidoc/web.adoc [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+This part of the reference documentation covers Spring Framework's support for the
+presentation tier (and specifically web-based presentation tiers) including support
+for WebSocket-style messaging in web applications.
 
-Chapters cover the Servlet-based <<mvc,Spring MVC>> web framework including <<mvc-view,Views>>,
-<<mvc-cors,CORS>>, and <<websocket,WebSocket>> support.
+Spring Framework's own web framework, <<mvc,Spring Web MVC>>, is covered in the
+first couple of chapters. Subsequent chapters are concerned with Spring Framework's
+integration with other web technologies, such as <<jsf,JSF>>.
```
Comment (sdeleuze) [inline src/docs/asciidoc/web.adoc]: Integration with other web technologies has been moved to another document so I am not sure this change is relevant.

File: src/docs/asciidoc/web.adoc [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
 
-Note that as of Spring Framework 5.0 web applications can also run on a
-<<spring-web-reactive, reactive web stack>> (Reactive Streams API + non-blocking runtime).
+The section then concludes with comprehensive coverage of the Spring Framework
```
Comment (sdeleuze) [inline src/docs/asciidoc/web.adoc]: All links except "reactive web stack" are good, so these changes are not relevant, as stated previously web and reactive web documentation are now splitted.

File: src/docs/asciidoc/web/webflux-functional.adoc [hunk 5]
```diff
@@ -108,10 +108,9 @@ a completion signal when the person has been read from the request and stored. S
 when the `Person` has been saved.
 <3> `getPerson` is a handler function that returns a single person, identified via the path
 variable `id`. We retrieve that `Person` via the repository, and create a JSON response if it is
-found. If it is not found, we use `switchIfEmpty(Mono<T>)` to return a 404 Not Found response.
+found. If it is not found, we use `otherwiseIfEmpty(Mono<T>)` to return a 404 Not Found response.
```
Comment (sdeleuze) [inline src/docs/asciidoc/web/webflux-functional.adoc]: `otherwiseIfEmpty` is deprecated, `switchIfEmpty` is the one to use.

File: src/docs/asciidoc/web/webflux.adoc [hunk 6]
```diff
@@ -1,12 +1,66 @@
-[[webflux]]
-= Spring WebFlux
+[[web-reactive]]
```
Comment (sdeleuze) [inline src/docs/asciidoc/web/webflux.adoc]: Already included in reactive-web.

**Review by sdeleuze — COMMENTED** [review]
Thanks for your proofreading but I am afraid most changes proposed are not relevant, see my various comments, so I am going to reject this PR. Feel free to comment if I missed some points.

**Comment by sdeleuze** [comment]
Thanks for this PR, but I have to close it without merging since these changes do not seem relevant to me.

**Comment by rstoyanchev** [comment]
@Buzzardo keep in mind the current WebFlux chapter was a temporary solution and is about to be replaced based on https://jira.spring.io/browse/SPR-15149. Given that there is probably not much value in editing it at this time.

**Pull request closed without merging**